setup.cfg

```diff
@@ -36,7 +36,7 @@ include_package_date = True
 packages = rtree
 
 [options.package_data]
-rtree = lib
+rtree = lib, py.typed
```

Comment:
It looks like the modifications to `setup.cfg` aren't needed. You can confirm this by running `python -m build` (assuming that build is installed) and checking `Rtree.egg-info/SOURCES.txt` to see what gets included in the wheel.

Reply:
Sounds good - removed.